approved/j750/prb1_flow.txt

```diff
@@ -119,22 +123,26 @@ DFF 1.1	Flow Table
 						Test	meas_ps_leakage	meas_ps_leakage	1170		97		6	Fail														
 						logprint	Speed_binning_example_bug_from_video_5																					
 						Test	test200_1							None		g200_7AF57D1_FAILED_0												
-						Test	test200_2							None		g200_7AF57D1_FAILED_1												
-						Test	test200_3							None		g200_7AF57D1_FAILED_2												
 						flag-true	g200_7AF57D1_FAILED																flag-true	g200_7AF57D1_FAILED_0				
+						Test	test200_2							None		g200_7AF57D1_FAILED_1												
 						flag-true	g200_7AF57D1_FAILED																flag-true	g200_7AF57D1_FAILED_1				
+						Test	test200_3							None		g200_7AF57D1_FAILED_2												
 						flag-true	g200_7AF57D1_FAILED																flag-true	g200_7AF57D1_FAILED_2				
-						Test	test100_1				5			Fail	g100_7AF57D1_RAN_0	g100_7AF57D1_RAN_0							flag-true	g200_7AF57D1_FAILED				
-						Test	test100_2				5			Fail	g100_7AF57D1_RAN_1	g100_7AF57D1_RAN_1							flag-true	g200_7AF57D1_FAILED				
-						Test	test100_3				5			Fail	g100_7AF57D1_RAN_2	g100_7AF57D1_RAN_2							flag-true	g200_7AF57D1_FAILED				
-						flag-true	g100_7AF57D1_RAN																flag-true	g100_7AF57D1_RAN_0				
-						flag-true	g100_7AF57D1_RAN																flag-true	g100_7AF57D1_RAN_1				
-						flag-true	g100_7AF57D1_RAN																flag-true	g100_7AF57D1_RAN_2				
+						flag-true	g100_7AF57D1_RAN																flag-true	g200_7AF57D1_FAILED				
+						Test	test100_1				5			Fail									flag-true	g200_7AF57D1_FAILED				
+						Test	test100_2				5			Fail									flag-true	g200_7AF57D1_FAILED				
+						Test	test100_3				5			Fail									flag-true	g200_7AF57D1_FAILED				

```

Comment:
This is a change to how if_ran is handled when it is applied to a group.
Previously the condition was applied to all contained tests, it is now far more simple and just sets the RAN flag immediately upon entering the group.